core/client-internal/src/test/java/tachyon/client/file/BaseFileSystemTest.java

```diff
@@ -69,7 +69,6 @@
    */
   @Before
   public void before() {
-    ClientContext.reset();
```

Comment:
that line actually modifies `mFileSystem`, not any of the contexts, so it should be ok